### ISSUE TITLE
The mixer is stuck

### ISSUE THREAD

**Issue by lolandhold** [decred/decrediton#3216]
The logs say 
```
GRPC: Streaming method /walletrpc.AccountMixerService/RunAccountMixer invoked by...
```
The mixer stays stuck after this log message.

**Comment by alexlyp** [comment]
This issue has been resolved for the upcoming patch release.  If you currently not able to mix, you may sweep you funds by going to On-Chain Transactions and selecting Send to Another Account and then select Unmixed for both then send yourself the full amount.  Once that transaction is confirmed, the mixer will once again work.

**Comment by amassarwi** [comment]
Also, it takes the mixer ~5 minutes to show any logs after invoking the RunAccountMixer RPC.

**Comment by lolandhold** [comment]
The mixer worked after the transaction that sweeps the funds was confirmed.

A constant amount of funds is stuck in the unmixed account and doesn't get mixed. Is this normal?

**Comment by lolandhold** [comment]
The logs say the wallet is locked.  Maybe this is why it's not mixing.
```
2021-02-20 20:26:03.011 [ERR] TKBY: wallet.MixAccount: wallet locked:: wallet.MixOutput(xxxxxxxx:31):: cspp.Confirm: the unmixed account
```

**Comment by alexlyp** [comment]
Resolved in 1.6.1
